subprojects/kotlin-dsl-provider-plugins/src/main/kotlin/org/gradle/kotlin/dsl/provider/plugins/precompiled/tasks/GeneratePrecompiledScriptPluginAccessors.kt

```diff
@@ -118,6 +120,14 @@ abstract class GeneratePrecompiledScriptPluginAccessors @Inject internal constru
     @get:Input
     abstract val strict: Property<Boolean>
```

Comment:
:question: Shall we start nagging about not using the strict mode in 7.0 already?